main.tf

```
@@ -449,6 +449,10 @@ resource "aws_ecs_task_definition" "main" {
 
   container_definitions = var.container_definitions == "" ? local.default_container_definitions : var.container_definitions
 
+  ephemeral_storage {
+    size_in_gib = var.ephemeral_storage_size_in_gib
+  }
+
```

I would recommend coding this so that if someone doesn't set the variable (which is most cases), then this doesn't get set. This will allow it to use the default value even if AWS increases it from 21GB which they sometime do when the Fargate version changes.

So you can do that by setting the var.ephemeral_storage_size_in_gib `default  = null` and then:

```suggestion
  dynamic "ephemeral_storage" {
    for_each = var.ephemeral_storage_size_in_gib != null ? [1] : [0]
    content {
      size_in_gib = var.ephemeral_storage_size_in_gib
    }
  }
```